webrtc/videoroom/assets/src/room.ts

```diff
@@ -89,6 +90,8 @@ const setup = async () => {
             removeVideoElement(track, stream);
           }
         },
+        onServerReplaceStream: replaceStream,
```

Comment:
Do we need `Server` in callback name?

Reply:
The `Server` word is to inform that this callback will be invoked on server indication. In other words there will be situations when server will tell us to replace stream. I think that it's a little clearer in this form but don't insist.

Reply:
I'm wondering if this is necessary here as this callback is exposed to UI so it can react to stream change, it should not care how this happens.